static/styles/zulip.css

```diff
@@ -1039,6 +1042,12 @@ td.pointer {
         word-wrap: unset;
     }
 
+    > .respond_button {
+        &:hover {
+            color: hsl(200, 100%, 40%);
+        }
+    }
```

Comment:
This very likely needs similar logic in `dark_theme.css`.

Reply:
I don't think this logic needs to be in `dark_theme.css` since the hover effect works as expected in the dark theme as well.
![hover](https://user-images.githubusercontent.com/66828942/216508010-f2b61b4e-d93a-4a98-a061-b70377860f06.gif)